editor/blocks/quote/index.js

```diff
@@ -0,0 +1,42 @@
+const Editable = wp.blocks.Editable;
+const { html } = wp.blocks.query;
+
+wp.blocks.registerBlock( 'core/quote', {
+	title: 'Quote',
+	icon: 'format-quote',
+	category: 'common',
+
+	attributes: {
+		value: html( 'blockquote > p' ),
```

Comment:
A `blockquote` could contain multiple paragraphs. In my explorer demo I use `text: query( 'p', html() )` instead which assigns `text` as an array of each paragraph's HTML.

https://aduth.github.io/hpq/